examples/src/main/java/org/neo4j/docs/driver/PassBookmarkExample.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.docs.driver;
+
+// tag::=pass-bookmarks-import[]
```

Comment:
Is `=` needed in `tag::=pass-bookmarks-import[]` and `end::=pass-bookmarks-import[]`?

Reply:
Company names are owned by @technige and I'm not sure how I ended up with those `=` signs :)